user/migrations/0001_initial.py

```diff
@@ -72,7 +75,9 @@ class Migration(migrations.Migration):
                 (
                     "email",
                     models.EmailField(
-                        blank=True, max_length=254, verbose_name="email address"
+                        blank=True,
+                        max_length=254,
+                        verbose_name="email address",
                     ),
```

Comment:
Ensure the `email` field is set as unique to serve as the unique identifier for the `User` model.